op-e2e/actions/sync_test.go

```diff
@@ -830,6 +832,264 @@ func TestELSyncTransitionstoCL(gt *testing.T) {
 	require.Equal(t, uint64(23), id.Number)
 }
 
+func TestELSyncTransitionsToCLSyncAfterNodeRestart(gt *testing.T) {
```

Comment:
If the new tests are mostly copied over from another test, it would be nice to actually only define one function that contains most testing logic and then add some parameterization, to achieve deduplication.